db/seeds/01_alta.rb

```diff
@@ -261,13 +261,19 @@
   )
 
   10.times do
-    AdopterApplication.create!(
+    adopter_application = AdopterApplication.new(
       notes: Faker::Lorem.paragraph,
       profile_show: true,
-      status: rand(0..5),
-      adopter_account: AdopterAccount.joins(:user).where(users: {organization_id: @organization.id}).sample,
+      status: rand(0..4),
+      adopter_account: AdopterAccount.all.sample,
       pet: Pet.all.sample
     )
+
+    if adopter_application.valid?
+      adopter_application.save!
+    else
+      redo
+    end
```

Comment:
I think this method is fine for ensuring 10 adopter applications are generated at least, but it does have the one potential problem where if it's not possible for 10 unique applications to exist, it will create an infinite loop. Another easy solution would be to just try 10 times and skip the invalid ones. Could also use a flag to only retry an iteration one time. I think this current one is fine though for a seed file.

Reply:
Yeah, Im in favor of avoiding infinite loops but agree this is fine for now